client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -390,7 +395,7 @@ public void write(PendingEvent event) {
             connection.send(new Append(segmentName, writerId, eventNumber, Unpooled.wrappedBuffer(event.getData()),
                                        event.getExpectedOffset()));
         } catch (ConnectionFailedException e) {
-            log.warn("Connection failed due to: ", e);
+            log.warn("Connection " + writerId + " failed due to: ", e);
```

Comment:
`  public void warn(String msg, Throwable t);`  this is the api being used. i.e. we do not have an api where we pass an argument with an Throwable.